Automattic/portkey-android
# Moved plus icon back to the stories library where it belongs

Fixes the plus icon being removed in #516 by mistake - and moving the resource from the example app to the stories library so it can be used in any host app

To test:
1. run the example app
2. add an image
3. observe the frame selector appears at the bottom and the adder control shows the plus icon correctly
